src/js/port_handler.js

```diff
@@ -39,18 +42,236 @@ PortHandler.check = function () {
     self.check_usb_devices();
     self.check_serial_devices();
 
+    self.check_elrs_devices();
+
     GUI.updateManualPortVisibility();
 
     setTimeout(function () {
         self.check();
     }, TIMEOUT_CHECK);
 };
 
+PortHandler.check_elrs_devices = function () {
+    const self = this;
+    self.check_mdns_devices();
+
+    if (!self.elrs_lock) {
+        self.elrs_lock = true;
+
+        setTimeout(() => {
+            ping.sys.probe('10.0.0.1', function (isAlive) {
+                let currentPorts = [{
+                    path: `tcp://10.0.0.1`,
+                    displayName: 'elrs_rx.local',
+                    vendorId: 0,
+                    productId: 0,
+                }];
+                if (isAlive && !self.initialPorts.find(p => p.path.endsWith('10.0.0.1'))) {
+                    setTimeout(() => {
+
+                        serial.connectTcp('10.0.0.1', null, null, function(connected) {
+                            if (connected !== false) {
+                                if (!self.initialPorts || self.initialPorts.length === 0) {
+                                    currentPorts = self.updatePortSelect(currentPorts);
+                                    self.selectPort(currentPorts);
+                                    self.initialPorts = currentPorts;
+                                } else {
+                                    self.removePort(currentPorts, false);
+                                    self.detectPort(currentPorts);
+                                }
+                            }
+
+                            serial.disconnect();
+
+                            self.elrs_lock = false;
+                        });
+                    }, 1000);
+                } else if(!isAlive && self.initialPorts.find(p => p.path.endsWith('10.0.0.1'))) {
+                    self.removePort(self.initialPorts.filter(p => !p.path.endsWith('10.0.0.1')), false);
+
+                    self.elrs_lock = false;
+                } else {
+                    self.elrs_lock = false;
+                }
+            });
+        }, 2000);
```

Comment:
Seems rather high. This function is already called every 500ms. So it has to fit into this time window.

Reply:
I don't want to flood the network. My second idea was to enable AP mode scanning my hand and don't have it on all the time. How do you think about that?

Reply:
That might be a better approach.

Reply:
ping is removed